source/lib/thunderbird.js

```diff
@@ -14,26 +20,10 @@ function showSimpleNewMessageNotification(isRSS) {
     var title = isRSS ? _("New_article") : _("New_message");
     var text = _("Number_of_unread_messages") + " " + count;
 
-    var utils = require('./utils.js');
-
-    // if linux => doing unclickable notification, so without actionList
-    var system = require("sdk/system");
-    var sps = require("sdk/simple-prefs").prefs;
-    if (sps['engine'] == 1 && system.platform === "linux") {
-        var notifApi = require('./linux.js');
-        if (notifApi.notifyWithActions(utils.getIcon(), title, text, system.name, null, null))
-          return;
-    }
-
-    var notifications = require("sdk/notifications");
-    notifications.notify({
-        title: title,
-        text: text,
-        iconURL: utils.getIcon(),
-    });
+    showNotification(title, text);
 }
 
```

Comment:
I split `showNewMessageNotification` into `showNewRSSNotification` and `showNewEmailNotification`, since now they are completely different. Maybe they could be merged again sometime if we enable notification customization for RSS as well.